### PR TITLE
NXP-23440: initial checkin

### PR DESCRIPTION
**Pull request by troger**

### CODE REVIEW

**Review by kevinleturc — COMMENTED** [review]
👍

File: ftest/webdriver/itests.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<project name="smart-search-webdriver-tests" xmlns:nx="urn:nuxeo-build" xmlns:artifact="urn:nuxeo-artifact">
```
Comment (ataillefer) [inline ftest/webdriver/itests.xml]: Probably want to rename to `jsf-to-web-ui-tests`.

File: ftest/webdriver/pom.xml [hunk 2]
```diff
@@ -0,0 +1,145 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo.marketplace.jsf.to.web.ui</groupId>
+    <artifactId>parent</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
```
Comment (ataillefer) [inline ftest/webdriver/pom.xml]: `../../pom.xml` no?

File: ftest/webdriver/itests.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  <taskdef resource="org/nuxeo/build/antlib.xml" uri="urn:nuxeo-build" />
+  <taskdef resource="org/nuxeo/build/artifact/antlib.xml" uri="urn:nuxeo-artifact" />
+  <taskdef resource="net/sf/antcontrib/antlib.xml" />
+
+  <property name="out.dir" value="${maven.project.build.directory}" />
+  <property name="nuxeoctl.option.clid" value="" />
```
Comment (ataillefer) [inline ftest/webdriver/itests.xml]: Useless I believe

File: ftest/webdriver/src/test/java/org/nuxeo/ftest/jsf/to/web/ui/ITJSFToWebUITest.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    @After
+    public void after() {
+        RestHelper.cleanup();
+    }
+
+    @Test
+    public void testWebUIRedirect() {
+        getLoginPage().login("Administrator", "Administrator", EmptyPage.class);
```
Comment (ataillefer) [inline ftest/webdriver/src/test/java/org/nuxeo/ftest/jsf/to/web/ui/ITJSFToWebUITest.java]: Why not just `getLoginPage().login("Administrator", "Administrator");`?

File: marketplace/src/main/resources/package.xml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  <title>Nuxeo JSF To Web UI</title>
+  <description>
+    <p>The Nuxeo JSF to Web UI addon adds the ability to redirect JSF links to the Web UI.</p>
+  </description>
+  <home-page>https://doc.nuxeo.com/nxdoc/nuxeo-jsf-to-web-ui/</home-page>
+  <vendor>Nuxeo</vendor>
+  <installer restart="true" />
+  <uninstaller restart="true" />
+  <!-- <hotreload-support>false</hotreload-support> -->
+  <!-- <require-terms-and-conditions-acceptance>false</require-terms-and-conditions-acceptance> -->
```
Comment (ataillefer) [inline marketplace/src/main/resources/package.xml]: Remove comments?

**Review by ataillefer — APPROVED** [review]

**Comment by ataillefer** [comment]
:+1:

File: marketplace/src/main/resources/package.xml [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  <supported>true</supported>
+  <platforms>
+    <platform>server-@DISTRIB_VERSION@</platform>
+    <platform>server-@DISTRIB_VERSION@-*</platform>
+  </platforms>
+  <dependencies>
+    <package>nuxeo-web-ui</package>
+  </dependencies>
+  <license>LGPL</license>
+  <license-url>http://www.gnu.org/licenses/lgpl.html</license-url>
```
Comment (efge) [inline marketplace/src/main/resources/package.xml]: We don't use LGPL anymore, but Apache License v2.0.

File: marketplace/pom.xml [hunk 7]
```diff
@@ -0,0 +1,41 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo.marketplace.jsf.to.web.ui</groupId>
+    <artifactId>parent</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
```
Comment (efge) [inline marketplace/pom.xml]: Why not start at 1.0.0-SNAPSHOT?

Comment (troger) [reply marketplace/pom.xml]: 1.0.0-SNASPHOT will be for 8.10,
1.1.0-SNAPSHOT for 9.10.

**Pull request merged**